README.md

```diff
@@ -52,6 +52,24 @@ Widgets Naming
 
 Where data source could be git, gerrit, mbox, etc.
 
+Widgets Title Naming
+--------------------
+
+In general, titles should follow the same naming scheme as the widget itself, substituting underscore characters by single spaces and adding capital letters:
+
+```
+ <data_source> <metric_used>
+ 
+ Example: Git Top Authors
+ Example: Git Authors
```

Comment:
I'd say that we only specify the data source of there is any chance of confusion as you mention for the data status panel.
So we basically use top authors or authors instead of git top authors in the title.